api/handlers.go

```diff
@@ -0,0 +1,222 @@
+package api
```

Comment:
Please keep student and marks APIs in separate files.
`students.go` and `marks.go`